pixeltable/catalog/table_version.py

```diff
@@ -53,6 +53,7 @@ class TableVersion:
     """
 
     id: UUID
+    dir_id: UUID
```

Comment:
dont think this is good place to cache it since it can change may be we dont need dir id in md, it doesnt tell much about table itself

Reply:
Yeah, the dir id isn't relevant here. It only matters in the context of Table (and subclasses).